Create Ethereum guides (#8757)
ethereum/ethereum-org-website: src/content/guides/how-to-use-a-wallet/index.md

---
title: How to use a wallet
description: A guide explaining how to send, receive tokens and connect to web3 projects.
lang: en
---

> # How to: Use a wallet

Reviewer: ```suggestion
# How to use a wallet
```

Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our [How to: "Register" an Ethereum account](/guides/how-to-register-an-ethereum-account/).

## Open your wallet

You should see a dashboard that will likely show your balance and contain buttons to send and receive tokens. 

## Receive cryptocurrency

Do you want to receive crypto into your wallet?

Each Ethereum account has its own receiving address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum addresses will always start with “0x”.

You need to provide the sender with your receiving address. Many wallet apps let you copy to clipboard the address or show a QR code to scan for easier usage. Avoid typing any Ethereum address manually. This is unsafe, and can easily lead to clerical errors and lost funds.

1. Open your wallet app.
2. Click on "Receive" or similarly worded option on the dashboard
3. Copy your Ethereum address to clipboard.
4. Provide the sender with your receiving Ethereum address.

> This is a general pattern, and different apps may vary or use different language. It should also be similar process if you are trying to send your funds from an exchange. 

Reviewer 2: Propose to move this above the bullet points prior as the transition from "avoid manual typing" to the steps is rather sudden. Suggested format below:

The steps listed below is a general pattern for receiving cryptocurrency, and different apps may vary or use different language. It should also be _a_ similar process if you are trying to send your funds from an exchange.

1. Open your wallet app.
2. Click on "Receive" or similarly worded option on the dashboard
3. Copy your Ethereum address to clipboard.
4. Provide the sender with your receiving Ethereum address.

## Send cryptocurrency

Would you like to send ETH to another wallet? 

1. Prepare the receiving address and the network of the recipient.
2. Open your wallet app.
3. Click on a “Send” button on the dashboard of your wallet or similarly worded alternative.  
4. Enter the receiving address or scan a QR code with your camera so that you don’t have to write the address manually.
<br/><br/>
![Send field for crypto address](./send.png)
<br/><br/>

> 5. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are, since these are not interchangeable.

Reviewer 2: Change "other side" to "recipient" for better clarity

```suggestion
5. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the recipient is using the same network as you are, since these are not interchangeable.
```

> 6. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 

Reviewer 2: Reworded to be more precise with the sequencing as users will usually be able to see the estimated fees prior to confirming the transaction. Moreover, some wallets will check the transaction total against balance and might not allow confirmation if there is insufficient funds to cover the estimated gas fee.

```suggestion
6. Ensure that your wallet has sufficient ETH to cover the transaction fee which varies depending on network conditions. Most wallets will automatically add the suggested fee to the transaction which you can then confirm.
```

> 7. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.

Reviewer 2: Minor language additions for clarity

```suggestion
7. Once your transaction is processed, the corresponding crypto amount will show up in the recipient’s account. This might take anywhere from a few seconds to a few minutes depending on [network transaction prioritisation](/developers/docs/gas/).
```

## Connecting to projects

Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.

1. Visit any project’s website.

> 2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 

Reviewer 2: Extended description for greater clarity

```suggestion
2. If the project's landing page is just a static description of the project, you should be able to click on an "Open the App" button in the menu which will navigate you to the actual web app.
```

3. Once you are in the app click on “Connect”

![Button allowing user to connect to the website with a wallet](./connect1.png)

> 4. Select your wallet from the offered list. If you can't see your wallet, it may be hidden under “WalletConnect” option.

Reviewer 2: Minor language change 

```suggestion
4. Select your wallet from the provided options list. If you can't see your wallet, it may be hidden under the “WalletConnect” option.
```

![Selecting from a list of wallets to connect with](./connect2.png)

> 5. Confirm a signature request in your wallet to start the connection. It should not cost anything.

Reviewer 2: Minor language edits

```suggestion
5. Confirm the signature request in your wallet to establish the connection. Do note that signing this message should not require spending any ETH.
```

### If I own an ETH address, do I own the same address on other blockchains?

You can use the same address on all EVM compatible blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.

### Can I use the same address on multiple devices?

Yes you can use the same address on multiple devices. Wallets are technically only an interface to show you your balance and to make transactions, your account isn't stored inside the wallet, but on the blockchain.

### I have not received the crypto, where can I check the status of a transaction?

You can use [block explorers](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/) to see the status of any transaction in real time. All you need to do is to search your wallet address or the ID of the transaction.

### Can I cancel or return transactions?

> No, once a transaction is initiated, you can not cancel the transaction.

Reviewer 2: Wondering if it should be made known to the user that an "initiated" tx can actually be cancelled as long as it's still unconfirmed. For example, sending a blank transaction consisting of the same nonce with a higher gas fee.

Alternatively, suggest to change "initiated" to "confirmed" to be more accurate.

Author: thank you for your submissions, also agree here. I will update the language

Reviewer 2: Sure thing, let me know if there are any particular pages or write-ups that you might need help on. Trying to see how best to contribute :)

---
title: How to use a wallet
description: A guide explaining how to send, receive tokens and connect to web3 projects.
lang: en
---

# How to: Use a wallet

> Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our How to: "Register" an Ethereum account.

Author: would removing the "how to" affect SEO? most of the users literary type "how to" when they ask for these guides which was my reasoning to structure it this way

<br/><br/>
<img src="./send.png" width="400" alt="Send field for crypto address"/>
<br/><br/>

4. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are. 
5. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
6. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.

## Connecting to projects

Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.

1. Visit any project’s website.
2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
3. Once you are in the app click on “Connect”

<br/><br/>
<img src="./connect1.png" width="400" alt="Button allowing user to connect to the website with a wallet"/>

> <br/><br/>

Author: @wackerow if i change this, will i loose the 400px width rule? The image is otherwise too big, should i resize it down first? or can we keep this way of adding it to the page?

4. Select your wallet from the offered list. If you can't see your wallet, maybe it will be hidden under “WalletConnect” option.

&nbsp;
<img src="./connect2.png" width="400" alt="Selecting from a list of wallets to connect with"/>
&nbsp;

5. Confirm a signature request in your wallet to start the connection. It should not cost anything.
6. Thats it! Start using the app. You can find some interesting projects on our [dApps page](https://ethereum.org/en/dapps/#explore).

> 
> <br/><br/>

Reviewer: ```suggestion
```

- You can use the same address on many popular blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.

**Can I use the same address on multiple devices?**

- Yes you can use the same address on multiple devices. Wallets are technically only an interface to show you your balance and to make transactions, your account isn't stored inside the wallet, but on the blockchain.

**I have not received the crypto, where can I check the status of a transaction?**

- You can use [block explorers](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/) to see the status of any transaction in real time. All you need to do is to search your wallet address or the ID of the transaction.

**Can I cancel or return transactions?**

- No, once a transaction is initiated, you can not cancel the transaction.

> ****

Reviewer: ```suggestion

```

---
title: How to use a wallet
description: A guide explaining how to send, receive tokens and connect to web3 projects.
lang: en
---

# How to: Use a wallet

Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our How to: "Register" an Ethereum account.

## Open your wallet

You should see a dashboard that will likely show your balance and contain buttons to Send and Receive tokens. 

## Receive cryptocurrency

Do you want to receive crypto into your wallet?

> Each wallet has its own receiving Ethereum address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum address will always start with “0x”.

Reviewer 3: An account isn't a wallet. A wallet can have many 'receiving addresses' 

Words are hard.... 😁

Author: This is targeted at absolute newcomers who cannot do this on their own which leads me to think its not the right timing to introduce the differences at this point.  I do address this later in FAQs as secondary information. Do you think thats confusing? @minimalsm

> You need to provide the sender with your receiving address. Many wallet apps will let you copy to clipboard the address or show a QR code to scan for an easier usage. 

Reviewer 3: ```suggestion
You need to provide the sender with your receiving address. Many wallet apps let you copy to clipboard the address or show a QR code to scan for easier usage. 
```

1. Open your wallet app.
2. Click on "Receive" or similarly worded option on the dashboard
3. Copy your Ethereum address to clipboard.
4. Provide the sender with your receiving Ethereum address.

> If you are trying to send yourself your funds from an exchange it is in principle the same process.

Reviewer 3: ```suggestion
It is the same process if you are trying to send your funds from an exchange.
```

> 

Reviewer 3: ```suggestion
```

## Send cryptocurrency

Would you like to send ETH to another wallet? 

1. Prepare the receiving address and the network of the recipient.
2. Open your wallet app.
3. Click on a “Send” button on the dashboard of your wallet or similarly worded alternative.  
4. Enter the receiving address or scan a QR code with your camera so that you don’t have to write the address manually.
<br/><br/>

> <img src="./send.png" width="400" alt="Send field for crypto address"/>

Reviewer 3: Adding this in the middle of a list will break the list semantics (and the numbering I think 🤔). We should have this image before or after the list.

Reviewer 3: Also, we should use markdown images here: [see example](https://ethereum.org/en/contributing/style-guide/#images)

Author: @minimalsm  yeah here i am a little bit conflicted on what to do. It would be great to have images as the steps go at contextually right places, but  it breaks the list. Having images after or before doesn't make sense. So its either as it is or none

## Connecting to projects

Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.

1. Visit any project’s website.
2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
3. Once you are in the app click on “Connect”

<br/><br/>
<img src="./connect1.png" width="400" alt="Button allowing user to connect to the website with a wallet"/>
<br/><br/>

4. Select your wallet from the offered list. If you can't see your wallet, maybe it will be hidden under “WalletConnect” option.

&nbsp;
<img src="./connect2.png" width="400" alt="Selecting from a list of wallets to connect with"/>
&nbsp;

5. Confirm a signature request in your wallet to start the connection. It should not cost anything.
6. Thats it! Start using the app. You can find some interesting projects on our [dApps page](https://ethereum.org/en/dapps/#explore).

<br/><br/>

### FAQ

**If I own an ETH address, do I own the same address on other blockchains?**

> - You can use the same address on many popular blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.

Reviewer 3: Maybe hinting at EVM compatibility here would be useful?

Something like 'many blockchains are built using the same structure as Ethereum (these are known as EVM compatible blockchain), you can use the same address on many of these blockchains.

Reviewer: Yeah, I see some reuse from above... Left a comment up there, but same would apply. A lot of it really does come down to the "structure" of the chain... it's not just that you used a mnemonic, and it's not just new-chain-new-addresss... It comes down to what type of address/signature scheme is used... most chains have their own setup, but those they went with the EVM share that with Ethereum.

**Can I use the same address on multiple devices?**

> - Yes you can use the same address on multiple devices. Wallets are technically only an interface to show you your balance and to make transactions, your wallets are not store inside the wallet, but on the blockchain.

Reviewer 3: > your wallets are not store inside the wallet, but on the blockchain.

your address/account isn't stored

**I have not received the crypto, where can I check the status of a transaction?**

- You can use [block explorers](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/) to see the status of any transaction in real time. All you need to do is to search your wallet address or the ID of the transaction.

**Can I cancel or return transactions?**

> - No, once transaction is initiated it cannot be canceled.

Reviewer 3: ```suggestion
- No, once a transaction is initiated, you can not cancel the transaction.
```

Reviewer 3: more caveats! technically you can cancel a transaction that is pending (e.g. if you sent too low a gas fee). however, if it's a successful transaction, then saying this makes more sense (let's ignore the complexities of that specific case here 😅)
